terratorch/cli_tools.py

```diff
@@ -101,6 +99,20 @@ def save_prediction(prediction, input_file_name, out_dir, dtype:str="int16"):
     logging.info(f"Saving output to {out_file_name} ...")
     write_tiff(result, os.path.join(out_dir, out_file_name), metadata)
 
+def import_custom_modules(custom_modules_path):
+    if custom_modules_path:
+        if os.path.isdir(custom_modules_path):
+
+            # Add 'custom_modules' folder to sys.path
+            workdir, module_dir = os.path.split(custom_modules_path)
```

Comment:
You may want a way of handling `:` separated paths. Ie if someone sets an environment variable with multiple paths, it is fairly standard to separate these with a `:`. Just a thought, otherwise you may want to throw an error for that case.

Reply:
I agree this is a topic to keep in mind.